framework/src/DriverFramework.cpp

```diff
@@ -171,7 +171,7 @@ timespec DriverFramework::offsetTimeToAbsoluteTime(uint64_t offset_time)
 	return ts;
 }
 
-//#if DF_ENABLE_BACKTRACE
+#if DF_ENABLE_BACKTRACE
```

Comment:
This change actually means that a lot of targets which do not define DF_LINUX will now not any more show a backtrace. @julianoes Can you validate Snapdragon?